tests/random_test.py

```diff
@@ -177,6 +178,22 @@ def testRngUniform(self, dtype):
       self._CheckCollisions(samples, jnp.finfo(dtype).nmant)
       self._CheckKolmogorovSmirnovCDF(samples, scipy.stats.uniform().cdf)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_dtype={}".format(np.dtype(dtype).name), "dtype": dtype}
+      for dtype in complex_dtypes))
+  def testRngUniform(self, dtype):
```

Comment:
The duplicate function name appears to be making flake8 sad; I suggest just adding `Complex` to the name. Same below.